src/pages/en.md

```diff
@@ -367,6 +366,22 @@ It's a good practice to publish RC/beta versions prior to releasing a major upda
 
 You can automate your release process by using, for example, [`gemsmith`](https://github.com/bkuhlmann/gemsmith) / [`gem-release`](https://github.com/svenfuchs/gem-release) or CI services (e.g. [Travis supports](https://docs.travis-ci.com/user/deployment/rubygems/) RubyGems deployments).
 
+- [ ] Don't let your gem bloat
+
+While it makes a great sense to add to your repository as more documentation and examples as possible, have a huge test suite and development
+guides, it's better not to include this unrelated data to the final package. Be reasonable about your gem content.
+The only essential parts, in the most cases, are basic README, license information, and the source code itself.
+
+Whitelisting gem content protects you from occasional and undesirable "additions" in the future.
+
+```ruby
+Gem::Specification.new do |spec|
+  # ...
+  spec.files = `git ls-files README.md LICENSE lib bin`.split
```

Comment:
Bundler generates `LICENSE.txt` by default, let's use it here, too:

```suggestion
  spec.files = `git ls-files README.md LICENSE.txt lib bin`.split
```

Reply:
Makes sense